sdk/wallet-adapter/packages/wallet-kit/src/utils/Dialog.tsx

```diff
@@ -30,6 +31,7 @@ export const Content = styled(Dialog.Content, {
   alignItems: "flex-end",
   padding: "$4",
   boxSizing: "border-box",
+  pointerEvents: "none!important",
```

Comment:
Curious why we need this

Reply:
The content component is above the Overlay and captures click events and clicking outside to close the modal doesn't work. So added this to allow clicks to be captured from the Overlay component. (And important because it is styled as `pointer-events: auto;` from the component implementation.

It's a bit hacky probably but the other way I tried makes more complicated the positioning of the dialog.